src/index.js

```diff
@@ -1,5 +1,9 @@
 module.exports = {
-  theme: {},
+  theme: {
+    debugOutline: true,
```

Comment:
Should we set default value for `debugOutline` to `false` ?

Reply:
Yes, that probably is a good idea.